include/cantera/numerics/Func1Factory.h

```diff
@@ -143,6 +143,13 @@ shared_ptr<Func1> newFunc1(const string& func1Type,
 shared_ptr<Func1> newFunc1(const string& func1Type,
                            const shared_ptr<Func1> f, double coeff);
 
+//! Check definition of functor object
+//! @param func1Type  String identifying functor type.
+//! @return integer flag indicating existence: '0' if undefined, '1' if @ref func1simple
+//!     or @ref func1advanced; '2' if @ref func1compound or '3' if @ref func1modified.
```

Comment:
Small suggestion to make the docstring a little more clear:

```suggestion
//! @return integer flag indicating existence: '0' if undefined; '1' if @ref func1simple
//!     or @ref func1advanced; '2' if @ref func1compound; '3' if @ref func1modified.
```

Separately, this is kind of another magic number, does it make sense to return strings here or an enum type?

Reply:
Fair point. I switched to an `enum` and marked the function as `@internal` (i.e. not intended to be made available at high-level APIs)